fuel-relayer/src/finalization_queue.rs

```diff
@@ -208,8 +209,9 @@ impl FinalizationQueue {
         }
         let last_diff = self.pending.back_mut().unwrap();
         match fuel_event {
-            EthEventLog::AssetDeposit(deposit) => {
-                last_diff.assets.insert(deposit.deposit_nonce, deposit);
+            EthEventLog::DaMessage(message) => {
+                let msg = DaMessage::from(&message).lock();
+                last_diff.messages.insert(*msg.id(), msg);
             }
             EthEventLog::Deposit { .. } => {
```

Comment:
Is this `Deposit` in the enumeration necessary anymore? I'm not sure why it was there in the old deposit/withdrawal setup either since `AssetDeposit` already existed

Reply:
https://github.com/FuelLabs/fuel-core/issues/515